examples/backends/sample/launch/agg_diffusion.sh

```
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Aggregated serving with the sample diffusion (raw-media) backend.
+# Exercises the DiffusionEngine path (frontend -> RawEngineAdapter ->
+# SampleDiffusionEngine) end-to-end on CPU — no GPU, no model.
+# GPUs: 0 (CPU-only)
+
+set -e
+trap 'echo Cleaning up...; kill 0' EXIT
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/launch_utils.sh" # print_launch_banner, wait_any_exit
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)